canonical/labels
# Add A11y review labels

## Done
Added the "Review: A11y" labels to the review set to indicate that the author of the change would like someone to perform an A11y specific review of it.

## QA
- Check the code looks right
- If you want to, check out this branch and follow the steps in the README to apply these labels to a dummy repo
- Check the labels have been added to the team information: https://discourse.canonical.com/t/labels-descriptions/151
